paddle/phi/kernels/fusion/onednn/fused_elementwise_kernel.cc

```diff
@@ -66,8 +66,18 @@ void FusedElementwiseKernel(const OneDNNContext& dev_ctx,
     std::swap(non_const_x, non_const_y);
   }
 
-  const auto src_x_memory = handler.AcquireSrcMemory(non_const_x);
-  const auto src_y_memory = handler.AcquireSecondSrcMemory(non_const_y);
+  const auto src_x_memory =
+      handler.swin_case ? (x.numel() == y.numel()
+                               ? handler.AcquireExtendSrcMemory(non_const_x, 0)
```

Comment:
what is extendsrc for?

Reply:
This aligns with former PR https://github.com/PaddlePaddle/Paddle/pull/59421. Since we need to manually broadcast src1/src2, I hereby name such operations as "extend".